### PR TITLE
Story/quad 180

### PR DESCRIPTION
**Pull request by yyk16293**

### CODE REVIEW

File: Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-settings.jsp [hunk 1]
```diff
@@ -8,27 +8,30 @@
 <script
 	src="${pageContext.servletContext.contextPath}/resources/txt-layout/js/jstree.min.js"></script>
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-settings.jsp]: I don't think we need this scrip anymore, do we?

File: Quadriga/src/main/webapp/WEB-INF/views/settings/addpublicpage.jsp [hunk 2]
```diff
@@ -6,24 +6,25 @@
...
 
 	function performAction(obj) {
 		var data = {};
 		var publicpageid = $(obj).closest('div.publicpageform').find('.publicpageid').val();
-		var title = $(obj).closest('div.publicpageform').find('.titleValue').val();
-		var desc = $(obj).closest('div.publicpageform').find('.descValue').val();
-		var order = $(obj).closest('div.publicpageform').find('.orderValue').val();
-		var linkTo = $(obj).closest('div.publicpageform').find('.linkTo').val();
-		var linkText = $(obj).closest('div.publicpageform').find('.linkText').val();
+		var title = $(obj).closest('div.publicpageform').find('#title').val();
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/settings/addpublicpage.jsp]: ids should be unique, so you can't reuse them in each about text block.

File: Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-settings.jsp [hunk 3]
```diff
@@ -8,27 +8,30 @@
...
+    <a id = "mainPageTab" class = 'list-group-item ${currentPage != "settings" ? "disabled" : ""}'
+			  href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${project.projectId}/settings"><i class="fa fa-cog"></i> Main Page Settings</a>
    
-<br>
-<a
-	href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${project.projectId}/settings/editabout"><i
-	class="fa fa-info-circle"></i> About Page</a>
-<br>
-<a
+    <a id = "aboutTab" class = 'list-group-item ${currentPage != "editabout" ? "disabled" : ""}'
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-settings.jsp]: and where is currentPage defined?

Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-settings.jsp]: as I said before, please take a look at how the main navigation tab selection works and do it here accordingly.

**Comment by jdamerow** [comment]
Please reopen **this** pull request when you're done with the changes, so that we can keep track of all the comments and changes.

File: Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-settings.jsp (hunk 1 above)
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-settings.jsp]: this is still here.

File: Quadriga/src/main/webapp/WEB-INF/views/settings/addpublicpage.jsp (hunk 2 above)
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/settings/addpublicpage.jsp]: these are still all ids

Comment (yyk16293) [inline Quadriga/src/main/webapp/WEB-INF/views/settings/addpublicpage.jsp]: I missed committing this file. I am checking on that and will update pull request

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
image
This needs to be the other way around. When you're on the about settings page, the other ones should be enabled and the about page settings page is disabled. Otherwise it doesn't make sense. And you shouldn't be able to click on the disabled entries.

**Comment by jdamerow** [comment]
ok to test

**Pull request merged**